src/renderer/atlas/AtlasEngine.api.cpp

```diff
@@ -659,6 +648,16 @@ void AtlasEngine::_resolveFontMetrics(const FontInfoDesired& fontInfoDesired, Fo
         BOOL exists = false;
         THROW_IF_FAILED(fontCollection->FindFamilyName(fontName.c_str(), &index, &exists));
 
+        // In case of a portable build, the given font may not be installed and instead be bundled next to our executable.
+        if constexpr (Feature_NearbyFontLoading::IsEnabled())
+        {
+            if (!exists && _updateWithNearbyFontCollection())
+            {
+                fontCollection = _api.s->font->fontCollection;
+                THROW_IF_FAILED(fontCollection->FindFamilyName(fontName.c_str(), &index, &exists));
+            }
+        }

```

Comment:
The fix for the dialog.

Reply:
okay so now, on the first failed font in the list, we'll fall back to the nearby font collection for all the remaining ones